app/frontend/styles/application.scss

```diff
@@ -13,6 +13,7 @@ $govuk-image-url-function: frontend-image-url;
 $govuk-breakpoints: (
   mobile:  320px,
   tablet:  641px,
+  filter_hide: 945px,
```

Comment:
We should change this to a name that is not as tied to the one usecase here... I'm trying to think of something.

Reply:
```suggestion
  large: 945px,
```

Reply:
Yeah - I had the same thought but couldn't think of one 🤔 Will change to this.